src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.InputHelper.cs

```diff
@@ -45,7 +49,28 @@ public static void Hold(UIElement element)
 
 			public static void Tap(UIElement element)
 			{
-#if !NETFX_CORE
+#if NETFX_CORE || __SKIA__
+				Finger finger = null;
+				MUXControlsTestApp.Utilities.RunOnUIThread.Execute(() =>
+				{
+					finger = InputInjector.TryCreate()?.GetFinger() ?? throw new InvalidOperationException("Failed to create finger");
+					var topLeft = element.TransformToVisual(Window.Current.Content).TransformPoint(new Point(0, 0));
+					var center = new Point(topLeft.X + element.RenderSize.Width / 2, topLeft.Y + element.RenderSize.Height / 2);
+					finger.Press(center);
+				});
+
+#if NETFX_CORE
+				// On Windows, We need to wait a bit before releasing for the popup to open.
+				Thread.Sleep(600);
```

Comment:
@ramezgerges comments sounds weird considering the filename. Should we accept a delay as parameter on the method instead ?